src/card.tsx

```diff
@@ -596,19 +596,15 @@ export const MediaCard = React.forwardRef<HTMLDivElement, MediaCardProps>(
                                 />
                             </div>
                             {asset && (
-                                <Box
-                                    className={styles.mediaCardAsset}
-                                    paddingLeft={{mobile: 16, desktop: 24}}
-                                    paddingTop={{mobile: 16, desktop: 24}}
-                                >
+                                <div className={styles.mediaCardAsset}>
                                     <div
```

Comment:
perhaps these 2 nested divs could be implemented as:
```jsx
<Box paddingX paddingY>
    <div style={applyCssVars(...)} className={styles.mediaCardAsset}>
        ...
    </div>
</Box>
```

Reply:
or even a single one

Reply:
I like the single div option, done